api/python/provisioner/commands/cluster_id.py

```diff
@@ -21,22 +21,77 @@
 import uuid
 from typing import Type
 
+from . import (
+     CommandParserFillerMixin
+)
 from .. import inputs
 from ..api import grains_get
-from ..config import LOCAL_MINION
+from ..config import (
+     LOCAL_MINION,
+     ALL_MINIONS
+)
+from ..vendor import attr
 
 from provisioner.commands import (
-     PillarSet,
-     GetClusterId
+     PillarGet,
+     PillarSet
 )
-from . import CommandParserFillerMixin
-
-from ..vendor import attr
 from provisioner.salt import local_minion_id
 
 logger = logging.getLogger(__name__)
 
 
+@attr.s(auto_attribs=True)
+class GetClusterId(CommandParserFillerMixin):
```

Comment:
Convert the class to `ClusterId`
No parameter mean get. Let's define it to be inline with properties in Python.